### DISCUSSION TITLE
https://onionui.github.io/ and Reddit / Discord links

### DISCUSSION THREAD

**Discussion by Amiga500** [OnionUI/Onion#1199] [Ideas]
In my opinion, Reddit and Discord links are very important, and not so visible like the github link one on the top right panel of https://onionui.github.io/. 

Can i suggest to think to put these two links near the github link? I know that they are listed in the bottom page, but probably deserve more visibility

**Comment by Aemiii91** [comment]
The Reddit and Discord server is not "by us". So, I'd say no, they don't belong in the top menu.

**Reply by Amiga500** [reply]
oh i understood! onionui.github.io rocks! I love it
